src/tree_cmd.cpp

```diff
@@ -720,11 +721,7 @@ static void TileLoop_Trees(TileIndex tile)
 
 	if (_settings_game.construction.extra_tree_placement == ETP_NO_GROWTH_NO_SPREAD) return;
 
-	if (GetTreeCounter(tile) < 15) {
-		AddTreeCounter(tile, 1);
-		return;
-	}
-	SetTreeCounter(tile, 0);
+	if ((cycle & 15) < 15) return;
```

Comment:
I would use `!= 15` instead of `< 15`. Both because that would be consistent with the previous if-statement, and because the `<` kind of implies there might be more than one case when there is onlyy one case.